ibm/resource_ibm_cis_healthcheck.go

```diff
@@ -24,6 +24,9 @@ const (
 	cisGLBHealthCheckAllowInsecure   = "allow_insecure"
 	cisGLBHealthCheckCreatedOn       = "create_on"
 	cisGLBHealthCheckModifiedOn      = "modified_on"
+	cisGLBHealthCheckHeader          = "header"
```

Comment:
`headers` would be more readable that `header`

Reply:
done